packages/manager/modules/telecom-spare/src/modems/order/modems-order.html

```diff
@@ -0,0 +1,210 @@
+                                <h3 data-translate="modems_order_shipping_mode_title"></h3>
+                                <tuc-shipping-mode-selection data-ng-model="$ctrl.order.shipping.mode"
+                                    data-tuc-shipping-mode-selection-relay="$ctrl.order.shipping.relay"
+                                    data-tuc-shipping-mode-selection-options="$ctrl.order.shipping.options">
+                                </tuc-shipping-mode-selection>
+                            </div>
+                        </div>
+                        <div class="col-sm-6">
+                            <h2 class="widget-presentation-title" data-translate="modems_order_modem_title"></h2>
+                            <div>
+                                <h3 data-translate="modems_order_select_title"></h3>
+                                <oui-select name="brand"
+                                            data-model="$ctrl.order.brand"
+                                            data-items="$ctrl.brands"
+                                            placeholder="{{ ::'modems_order_place_holder' | translate}}"
+                                            data-match="name">
+                                </oui-select>
+                                <div>
+                                    <label data-translate="modems_order_select_quantity"></label>
+                                    <oui-numeric data-model="$ctrl.quantity" data-max="50"></oui-numeric>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <button type="button" class="oui-button oui-button_primary oui-button_icon-right float-right mt-2"
+                        data-ng-disabled="($ctrl.quantity === 0) || !$ctrl.order.brand || !$ctrl.order.contact || ($ctrl.order.shipping.mode === 'mondialRelay' && !$ctrl.order.shipping.relay)"
+                        data-ng-click="$ctrl.changeStep('summary')">
+                        <span data-translate="modems_order_order_next"></span>
+                        <span class="oui-icon oui-icon_circle oui-icon-chevron-right" aria-hidden="true"></span>
+                    </button>
+                </section>
+
+                <!-- SUMMARY -->
+                <section class="widget-presentation" data-ng-if="$ctrl.orderStep === 'summary'">
+                    <h2 class="widget-presentation-title" data-translate="modems_order_summary_title"></h2>
+
+                    <div class="text-center" data-ng-if="$ctrl.isStepLoading">
+                        <oui-spinner></oui-spinner>
+                    </div>
+
+                    <div data-ng-if="!$ctrl.isStepLoading">
+
+                        <!-- PRICE SUMMARY TABLE -->
+                        <h3 data-translate="modems_order_summary"></h3>
+                        <div class="oui-datagrid-responsive-container">
+                            <table class="oui-datagrid">
```

Comment:
use a `oui-datagrid` or at least its style

Reply:
that is it done no? I used oui-datagrid style

Reply:
see means using a component :)

`<oui-datagrid ...>` :)

Reply:
Sorry but that is a little bit weird, I used oui-datagrid style, the table works fine, I don't recode all for the same result but with more tests to make to check that it continues to work fine.